cogs/secretSanta.py

```diff
@@ -0,0 +1,145 @@
+import nextcord
+from nextcord.ext import tasks, commands
+import time
+from datetime import *
+import random
+
+msg = ''
+msg_id = ''
+participatingList = []
+b = ''
+santas =[] 
+guild_id = 0
+messagedFlag = False
+
+class santa(commands.Cog):
+	def __init__(self, client):
+		self.client = client
+
+
+	@tasks.loop(seconds=10)
+	async def timeCheck(self):
+		global participatingList
+		global guild_id
+		global messagedFlag 
+
+
+		t = datetime.today()
+		firstPos = 0
+		
+		if t.month == 12:
+			if t.day == 18:
+				if t.hour == 00:
+					if t.minute == 00:
+						if participatingList != None:

```

Comment:
Change multiple if statements to a single if statement using `&&` or `and` operator.

There should not be so much indentation

Reply:
```suggestion
		if t.month == 12 && t.day == 18 &&  t.hour == 00 && t.minute == 00:
				if participatingList != None:

```

Reply:
If that's the case then you may reject the suggestion and keep it.

However, for future uses, try to stick with single line conditions. Python is sensitive to indentations and it can take one wrong indentation to change the output if an entire function. Single line if statements are just safer.

Reply:
No, just No, this is bad. if you want to do this, at least use guard clauses, this is just awful.